src/main/java/pl/javastart/task/ToDo.java

```diff
@@ -0,0 +1,70 @@
+package pl.javastart.task;
+
+public class ToDo {
+    String name;
+    String description;
+    int priority;
+    Person person;
+
+    public ToDo(String name, String description, int priority, Person person) {
+        this.name = name;
+        this.description = description;
+        this.priority = priority;
+        this.person = person;
+    }
```

Comment:
Ten fragment:
```
        this.name = name;
        this.description = description;
        this.priority = priority;
```
Jest już w ramach innego konstruktora. Zamiast powtarzać kod uruchom tamten konstruktor korzystając z `this`